code/modules/mob/mob_movement.dm

```diff
@@ -363,7 +363,7 @@
 	if(!Process_Spacemove(get_dir(pulling.loc, A)))
 		return
 	var/target_turf = get_step(pulling, get_dir(pulling.loc, A))
-	if(!Adjacent(target_turf)) //Make sure the turf we are trying to pull to is adjacent to the user.
+	if(get_dist(target_turf, loc) > 1) //Make sure the turf we are trying to pull to is adjacent to the user.
```

Comment:
would you leave a comment as to why the adjacent proc isnt used?

Reply:
Comment has been updated.